beacon_node/src/cli.rs

```diff
@@ -320,6 +320,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 and never provide an untrusted URL.")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("monitoring-endpoint-frequency")
+                .long("monitoring-endpoint-frequency")
+                .value_name("SECONDS")
+                .help("Defines how many seconds to wait between each message sent to \
+                       the monitoring-endpoint.")
+                .default_value(Box::leak(monitoring_api::DEFAULT_UPDATE_DURATION.to_string().into_boxed_str()))
```

Comment:
This is cool!

Reply:
Cool but unfortunately having a `default_value` and a `requires("monitoring-endpoint")` broke `clap` :(

So I dropped this